src/synthesized_insight/check.py

```diff
@@ -99,7 +99,7 @@ def continuous(self, sr: pd.Series) -> bool:
         """
         sr = self.infer_dtype(sr)
         sr_dtype = str(sr.dtype)
-        if len(sr.unique()) >= max(min(self.min_num_unique, len(sr)),
+        if len(sr.unique()) >= max(self.min_num_unique,
```

Comment:
This min requirement is a bit silly - it means that series like [1, 2, 3] are considered continuous. This might be fine but for the statistical tests that check
```
if self.continuous(sr_a) and self.continuous(sr_b):
    return True
if self.categorical(sr_a) and self.categorical(sr_b):
    return True
return False
```
It is quite important that we mark things as categorical or continuous correctly

Reply:
I think [1,2,3] or [1,2,3,4,5,6,7,8,9,10] should be considered continuous, however [1,2,2] / [1,3,2,3,4,5,6,3,2,7] might not be considered continuous. I think it is minor point so I'm okay either ways.